modules/bibupload/lib/bibupload.py

```diff
@@ -297,6 +297,21 @@ def bibupload(record, opt_mode=None, opt_notimechange=0, oai_rec_id="", pretend=
         # Update Mode
         # Retrieve the old record to update
         rec_old = get_record(rec_id)
+        # Check if the record has been migrated to a different instance
+        migrated = record_get_field_values(rec_old, '980', ' ', ' ', 'c')
```

Comment:
Keep in mind that we might have records with *something* in field 980__c that non necessarily means migrated.
There are few records on CDS like that:
```
In [2]: perform_request_search(p="980__c:/.*/")
Out[2]: [585509, 1119156, 1171956, 1355275, 1712225]
```
Can we make this condition configurable, so others can use it (otherwise, at least we can check explicitly for `MIGRATED` like in https://github.com/CERNDocumentServer/invenio/blob/prod/modules/websearch/lib/websearch_webinterface.py#L317) ?